cmd/swapcli/main.go

```diff
@@ -398,12 +398,8 @@ func runBalances(ctx *cli.Context) error {
 }
 
 func runDiscover(ctx *cli.Context) error {
-	provides, err := providesStrToVal(ctx.String(flagProvides))
-	if err != nil {
-		return err
-	}
-
 	c := newRRPClient(ctx)
+	provides := ctx.String(flagProvides)
```

Comment:
Do we need a `swapcli` feature to discover other relayers?

Reply:
if you pass the "isrelayer" string into the discover `provides` field, it'll return relayers. we can document that, I just don't expect it to be something users will use